ui-toolkit/modals.md

```diff
@@ -1,7 +1,148 @@
+
+<div class="content-50 content-first">
+  <h4 id="examples">Examples of when to use modals</h4>
+  <ul>
+    <li>Confirm an action that can’t be undone, such as “Are you sure you want to delete this?</li>
+    <li>Display a warning that something is about to expire or timeout</li>
+    <li>Save a form that requires inputs separate from the main workflow, like entering a “Save as” file name</li>
+    <li>Focus attention on the need to accept or acknowledge something before the user can proceed</li>
+  </ul>
+</div>
+<div class="content-50 content-last">
+{::nomarkdown}
+<img src="/design-manual/static/img/modals/formexpiring.png" /> 
+{:/nomarkdown}
+</div>
+
+<h2 id="behavior">Behavior<span class="cf-code-link">
+</span></h2>
+ 
+<h3 id="opening">Opening a modal</h3>
+<p>Modals can be triggered by links, buttons, or even inactivity on a site. When a modal opens, set the focus to the first element users need to interact with. This could be either the first form field in the modal window or the primary action button.</p> 
+
+<h3 id="scrolling">Scrolling within a modal</h3>
+<p>Modal windows should not scroll. If a modal contains enough information to require scrolling, use an alternative method of displaying content, such as a different UI element or another content page.</p>
+
+<h3 id="closing">Closing a modal</h3>
+<p>Actions inside modals should be clearly marked. There should be a clear path forward (“continue”) and back (“cancel”), in addition to a clearly labeled way to close the modal without taking any action at all. When a modal window is closed, return focus to the last active element on the page behind the modal.</p>
+
+<p>Modal windows are always closable by clicking or tapping:</p>
+  <ul>
+    <li>A delete minicon or label in the upper right corner</li>
+    <li>The next action button or link</li>
+    <li>Anywhere outside the modal window</li>
+    <li>ESC key on a keyboard</li>
+  </ul>
+
+<h3 id="accessible-behavior">Creating accessible behaviors</h3>
+<p>Keyboard access should be limited to only interacting with the modal dialog once it is visible.</p>
+  <ul>
+    <li>The close minicon needs to be accessible from a keyboard.</li>
+    <li>The escape key should also close the modal.</li>
+    <li>The enter key should should be mapped to any submission forms in the modal.</li>
+  </ul>
+
+<p>Provide separate focus and hover states for the close minicon and any “next” action buttons.</p>
+
+<p>The find function (ctrl+F) will not search information contained within a modal window.</p> 
+
+<p>Include offscreen instructions that describe the modal dialog and how to interact with it.</p>
+
+<h4 id="inactive">Specific development directions</h4>
+  <ul>
+    <li>Modal windows should be marked with aria-hidden="true" and toggled to false when visible and given the role=dialog ARIA role.</li>
```

Comment:
The first three bullet points here are a bit confusing. Maybe break them apart or rewrite them, so it's clearer what is to be done for each piece when the modal is visible or not visible (if that's the deciding factor)?

Also, add little ` tick marks around all the `aria-hidden="true"` and `role="dialog"` attributes.